ibis/expr/scope.py

```diff
@@ -122,10 +122,10 @@ def merge_scope(self, other_scope: 'Scope', overwrite=False) -> 'Scope':
         """
         result = Scope()
 
-        for op in self.items():
+        for op in self._items:
```

Comment:
should be `_items.keys()` no?

Reply:
I am not sure why using `.keys()` is better here, `_items` is a dict and `op` is its key. Isn't a simple `in` enough?

Reply:
because its not obvious that you _items is actually a dict from just looknig here, whereas .keys() is very explict

Reply:
OK, fixed it.